cmd/lambda/publish_lease_events/main.go

```diff
@@ -113,12 +113,18 @@ func handleRecord(input *handleRecordInput) error {
 		if didBecomeInactive {
 			// Before adding the account to any queues, make sure the account is
 			// updated to NotReady state.
-			acct, err := input.dbSvc.TransitionAccountStatus(lease.AccountID, db.Leased, db.NotReady)
+			_, err := input.dbSvc.TransitionAccountStatus(lease.AccountID, db.Leased, db.NotReady)
 			if err != nil {
 				log.Printf("ERROR: Failed to mark AccountStatus=NotReady for %s, after lease for %s became inactive",
 					lease.AccountID, lease.PrincipalID)
 			}
 
+			acct, err := input.dbSvc.GetAccount(lease.AccountID)
```

Comment:
Not a big deal, but you could probably throw this inside the `if err != nil {}` block, so it only does an additional query if the transition query fails

Reply:
Very true.  I'm going to fix it quick.  Saving DB calls is what I would like to do.